src/client/utils/prismjs/PrismCodeParser.js

```diff
@@ -0,0 +1,194 @@
+/* flow */
+
+import Prism from 'prismjs'
+
+const REMOVE_MARK = '-'
+const ADDED_MARK = '+'
+
+/* eslint-disable */
+
+export const getLineNumbers = (line) => {
```

Comment:
I think this function needs some form of annotation. I find it incredibly difficult to read.

It's not immediately clear to me what the assumptions about the format of the lines are, what the goal of of the two regexes are, what the input argument is, and what we are trying to achieve with the function.

*Edit*:
Actually this entire file has the same issues for me. This may be fine. But with some annotation, or some references to some documentation, or just a top comment describing what this does, I think it would be many many times easier to parse -- at least for me.